test/client/js/helpers/friendlysizeTest.js

```diff
@@ -5,14 +5,23 @@ const friendlysize = require("../../../../client/js/helpers/friendlysize").defau
 
 describe("friendlysize helper", function() {
 	it("should render big values in human-readable version", function() {
-		expect(friendlysize(51200)).to.equal("50 KB");
+		expect(friendlysize(51200)).to.equal("50 KiB");
+		expect(friendlysize(Math.pow(1024, 1))).to.equal("1 KiB");
+		expect(friendlysize(Math.pow(1024, 2))).to.equal("1 MiB");
+		expect(friendlysize(Math.pow(1024, 3))).to.equal("1 GiB");
+		expect(friendlysize(Math.pow(1024, 4))).to.equal("1 TiB");
+		expect(friendlysize(Math.pow(1024, 5))).to.equal("1 PiB");
 	});
 
 	it("should round with 1 digit", function() {
-		expect(friendlysize(1234567)).to.equal("1.2 MB");
+		expect(friendlysize(1234567)).to.equal("1.2 MiB");
 	});
 
 	it("should render special case 0 as 0 Bytes", function() {
-		expect(friendlysize(0)).to.equal("0 Bytes");
+		expect(friendlysize(0)).to.equal("0 B");
+	});
```

Comment:
From the title "it should render **special case** 0 as 0 Bytes" it sounds like it was very deliberate and it shouldn't change.

Reply:
It's a special case because Math.log(0) doesn't work, there's a `size>0` in the code for that.

Reply:
Oh.

As a user though, I think "Bytes" is probably clearer than "B", so perhaps that shouldn't change.

Reply:
That is a good point, I will change that.